Description: Fix "ambiguous name" error when testing as a separate crate
Bug: https://github.com/dtolnay/indoc/issues/58
--- a/src/lib.rs
+++ b/src/lib.rs
@@ -56,4 +56,4 @@
 
 mod unindent;
 
-pub use unindent::*;
+pub use crate::unindent::*;
